pymc3/math.py

```diff
@@ -71,9 +71,9 @@ def kron_vector_op(v):
         m = m[:, None]  # Treat 1D array as Nx1 matrix
     if m.ndim != 2:  # Has not been tested otherwise
         raise ValueError('m must have ndim <= 2, not {}'.format(mat.ndim))
-    m = m.T
-    res, _ = theano.scan(kron_vector_op, sequences=[m])
-    return res.T
+    res = kron_vector_op(m)
```

Comment:
nice!

Reply:
I'm not sure why I did not do this in the first place. Though I wanted to avoid scan if at all possible, I feel like there was certain cases where "nicer" ways failed, such as with non-square matrices or something, but I can't remember. In any case, if the tests pass and everything still works I like this a lot!